sw/device/silicon_creator/lib/error.h

```diff
@@ -21,21 +21,25 @@ extern "C" {
  * Choose a two-letter identifier for each module and encode the module
  * identifier the concatenated ASCII representation of those letters.
  */
+#define MODULE_CODE(ch0_, ch1_) ((ch0_) << 8 | (ch1_))
```

Comment:
note: This reverses the byte order when compared to previous scheme (just wanted to make sure that this was intended).

Reply:
This is intended.  The previous expression of the module codes as hex-constants was inconsistent - some where in opposite orders.

The ordering here causes the printed `boot_fault` code from `shutdown_finalize` to be in the natural human expected order.

Example: when the current mask ROM fails to find a ROM_EXT image in mask_rom.c:220:

```
RETURN_IF_ERROR(romextimage_manifest_get(kFlashSlotA, &manifest));
```

The error code is `kErrorRomextimageInternal`, which then gets printed as:

```
boot_fault: 0x0252450d
```

The middle two bytes 0x5245 are the module code. By way of asciitable.com, we see this module 'RE'.